### PR TITLE
Package v0.55.0

### PR DESCRIPTION
**Pull request by rram**
See the `README.debian` file for build instructions. Review on the review process also accepted.

:eyeglasses: @spladug @kjoconnor

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ libssl-dev,
+ libtool,
+ pkg-config,
+ python-dev
+
+Package: libfolly55
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: Facebook Open-source LibrarY
```
Comment (kjoconnor) [inline debian/control]: `LibrarY`

**Comment by kjoconnor** [comment]
:fish:

File: debian/control (hunk 1 above)
Comment (rram) [inline debian/control]: Was intentional, but looks like they got rid of it: https://github.com/facebook/folly/commit/b85957465746e9275528ceeb4d6646094a586874

**Comment by spladug** [comment]
lgtm! :fish:

**Comment by rram** [comment]
Gonna redo this one. Changing branch names

**Pull request closed without merging**